hudi-common/src/main/java/org/apache/hudi/common/util/queue/DisruptorMessageQueue.java

```diff
@@ -60,6 +61,10 @@ public long size() {
 
   @Override
   public void insertRecord(I value) throws Exception {
+    if (!isStart) {
```

Comment:
I don't think we need this, it would throw anyway if the queue has not been started yet

Reply:
> We assert in insertRecord that the queue is started

Here just assert the queue is started, in case others use this queue without calling the `start`

Reply:
@boneanxs we need to keep in mind that this lies on the hot-path of being executed for every record, so we should keep the bar really high in terms of what we allow to be executed for every record.

That being said, i also see your point of adding it to make sure it would fail definitively unless queue has been started

Reply:
Let's keep it, please rename it to `isStarted`